late/chroot_scripts/96-update_md5sum.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
```

Comment:
Instead of running as a late script, since this is pure python code I think it would make more sense to run it directly from bootstrap in Install (https://github.com/superm1/dell-recovery/blob/master/ubiquity/dell-bootstrap.py#L1254).

is there a reason this fails?  If not, please move it there.